code/modules/media/broadcast/transmitters/broadcast.dm

```diff
@@ -95,8 +95,6 @@
 
 /obj/machinery/media/transmitter/broadcast/attackby(var/obj/item/W, mob/user)
 	. = ..()
-	if(.)
```

Comment:
This would result in things like unconscious people and dead people being able to attack it with objects